source/index.rst

```diff
@@ -75,13 +75,15 @@ These are institutions who were early adopters or provided HPC resources for dev
   :caption: How-Tos
 
   how-tos/apps.rst
+  how-tos/app-development
+  how-tos/tutorials-interactive-apps
 
 
 .. toctree::
    :maxdepth: 2
    :caption: Extend
 
-   app-development/interactive/setup
+   how-tos/app-development/interactive/setup
```

Comment:
just remove this link if it's a child of `how-tos/app-development`.

Reply:
This one has me scratching me head. Removing it breaks the build, and for now I'd rather leave this and handle refactoring this section separately as it was not in scope but those sections needed moving for the other app-dev stuff.

Basically, let's make another ticket to refactor this section as well because the toctree is confusing and needs to be decyphered and fixed.